pom.xml

```diff
@@ -293,9 +204,6 @@
 							<goal>integration-test</goal>
 						</goals>
 						<configuration>
-							<includes>
```

Comment:
Instruction for **Running with Maven running** in README.md 

This project can be built with Apache Maven. The project uses Liberty Maven Plug-in to automatically download and install Liberty ~~profile~~ `with Java EE7 Web Profile `  runtime from the ~~Liberty repository~~ `Maven Central`. Liberty Maven Plug-in is also used to create, configure, and run the application on the Liberty server.